test/nxos/mocked_data/test_get_vlans/6000/expected_result.json

```diff
@@ -1,44 +1,7 @@
 {
   "230": {
     "name": "database",
-    "interfaces": [
```

Comment:
Could you please clarify why you're removing these?

I'm not familiar with this `vlanshowplist-ifidx`, from your description from #1145 I understand that `Using get_vlans for nxos_ssh fails when there are VLANs defined with no interfaces assigned.`, however in the mock data associated for this test case I understand that there are: https://github.com/napalm-automation/napalm/blob/8f986aaf4b2c1f7b8cbc5b27ac77ed78d7fdccbd/test/nxos/mocked_data/test_get_vlans/6000/show_vlan_brief.json#L17-L19... what's the secret key here? :smile:

Reply:
vlanshowplist-ifidx is the list of Interfaces associated to the VLAN. If there are no interfaces associated to the VLAN, vlanshowplist-ifidx doesn't exist (which creates the bug). 

In order to create a test case for this, I just modified one VLAN from the existing test data to remove vlanshowplist-ifidx. You're right ithat I could have added a VLAN instead, let me know if that is your preference.

Reply:
I see, yes, please add a separate test case, without adjusting the existing ones.